### PR TITLE
Editorial lsu

### PR DESCRIPTION
**Pull request by amilan17**

### CODE REVIEW

**Comment by amilan17** [comment]
@marijanacrepulja -- Most of these changes are purely editorial, but there are a couple things (above) that need some attention.

File: BUFRCREX_CodeFlag_en_33.csv [hunk 1]
```diff
@@ -637,7 +637,7 @@ FXY,ElementName_en,CodeFigure,EntryName_en,EntryName_sub1_en,EntryName_sub2_en,N
 033098,MWI data quality flag,All 9,Missing,,,,,Operational
 033099,MWI navigation status flag,1,Geolocation of channel is erroneous or degraded,,,,,Operational
 033099,MWI navigation status flag,2,Time sequence error,,,,,Operational
-033099,MWI navigation status flag,3,Missing or corrupted NAVATT file lead to the use of predicted orbit files,,,,,Operational
+033099,MWI navigation status flag,3,Missing or corrupted NAVATT file led to the use of predicted orbit files,,,,,Operational
```
[the remark this answers is not in the document]
Comment (marijanacrepulja) [reply BUFRCREX_CodeFlag_en_33.csv:640]: @SimonElliottEUM and @antoinemerle could you please confirm correct wording.

[the remark this answers is not in the document]
Comment (SimonElliottEUM) [reply BUFRCREX_CodeFlag_en_33.csv:640]: @antoinemerle @marijanacrepulja - yes "led" is correct and "lead" is wrong. "led" is the simple past participle of the verb "to lead".

(I know lots of people use "lead" as the past tense of "to lead". It is an easy mistake if you think too much about "to read" -> "read")

File: notes/acronyms.csv [hunk 2]
```diff
@@ -28,6 +28,7 @@ IASI-NG,Infrared Atmospheric Sounding Interferometer - New Generation
 ICI,Ice Cloud Imager
 IERS,International Earth Rotation Service
 IFOV,instantaneous field of view
+IFP,
```
[the remark this answers is not in the document]
Comment (marijanacrepulja) [reply notes/acronyms.csv:31]: @amilan17 
Intermediate Frequency Processor (IFP)

File: notes/acronyms.csv [hunk 3]
```diff
@@ -51,13 +52,13 @@ OBCT,on board calibration target
 OCOG,offset centre of gravity
 PCS,principle component scores
 PRF,pulse repetition frequency
-PRT,platinum resistance temperature
```
[the remark this answers is not in the document]
Comment (marijanacrepulja) [reply notes/acronyms.csv:54]: PRT | Platinum resistance thermometer

It is defined in https://github.com/wmo-im/BUFR4/issues/161

File: notes/acronyms.csv [hunk 4]
```diff
@@ -51,13 +52,13 @@ OBCT,on board calibration target
...
 PRT,platinum resistance thermometer
 PTR,pulse target response
 RAD,radiances
 RDR,raw data record
 RED,redundancy
 RF,radio frequency
+RFE,
```
[the remark this answers is not in the document]
Comment (marijanacrepulja) [reply notes/acronyms.csv:61]: @amilan17 

RFE = Receiver Front End

File: notes/acronyms.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (amilan17) [reply notes/acronyms.csv:54]: @marijanacrepulja @antoinemerle 
"PRT | Platinum resistance thermometer" was added to the acronym table, but there is the same acronym with a different meaning: "PRT | platinum resistance temperature". Are they both correct or is one of them wrong?

**Pull request merged**